neo/public/javascripts/map/controllers.js

```diff
@@ -55,7 +55,7 @@ angular.module('cloudberry.map', ['leaflet-directive', 'cloudberry.common'])
           dashArray: '',
           fillOpacity: 0.7
         },
-        colors: ['#053061', '#2166ac', '#4393c3', '#92c5de', '#d1e5f0', '#f7f7f7', '#fddbc7', '#f4a582', '#d6604d', '#b2182b', '#67001f'],
+        colors: ['#053061', '#2166ac', '#4393c3', '#92c5de', '#d1e5f0', '#fddbc7', '#f4a582', '#d6604d', '#b2182b', '#67001f'],
```

Comment:
we should also update the corresponding usage for this `colors`. like [`getColor`](https://github.com/ISG-ICS/cloudberry/pull/162/files#diff-0b388f44b244331841f2d0123c7791f1L263)